lib/bpk-appearance/src/withBpkAppearance.js

```diff
@@ -51,7 +51,11 @@ const getDisplayName = Component =>
  * @returns {Component} the wrapped component with an extra `bpkAppearance` prop.
  */
 const withBpkAppearance = <Config>(
-  Component: AbstractComponent<{| ...$Exact<Config>, ...InjectedProps |}>,
+  Component: AbstractComponent<{|
+    // $FlowFixMe
```

Comment:
What error is this suppressing? Our convention is to include the reason: https://github.com/Skyscanner/backpack/blob/master/decisions/ignoring.md